### PR TITLE
Node actions corresponding to RAP actions

### PR DESCRIPTION
**Pull request by HeikoTheissen**

### CODE REVIEW

File: vocabularies/Hierarchy.xml [hunk 1]
```diff
@@ -158,26 +156,14 @@ The template actions themselves cannot be invoked.
           <Annotation Term="Core.Description" String="Whether the service provides the [`StartNodeCounts`](#StartNodeCounts) instance annotation" />
         </Property>
         -->
-        <Property Name="CreateAction" Type="Core.QualifiedActionName" Nullable="true">
-          <Annotation Term="Core.Description" String="Action that creates a new node, following [this template](#Template_CreateAction)" />
+        <Property Name="LinkAction" Type="Core.QualifiedActionName" Nullable="true">
+          <Annotation Term="Core.Description" String="Action that creates a new node, following [this template](#Template_LinkAction)" />
```
Comment (ralfhandl) [inline vocabularies/Hierarchy.xml:160]: Adjust text to match property name

File: vocabularies/Hierarchy.xml [hunk 2]
```diff
@@ -158,26 +156,14 @@ The template actions themselves cannot be invoked.
...
         </Property>
-        <Property Name="CreateChildAction" Type="Core.QualifiedActionName" Nullable="true">
-          <Annotation Term="Core.Description" String="Instance-bound action that creates a new non-root node, following [this template](#Template_CreateChildAction)" />
+        <Property Name="UnlinkAction" Type="Core.QualifiedActionName" Nullable="true">
+          <Annotation Term="Core.Description" String="Action that creates a new node, following [this template](#Template_UnlinkAction)" />
```
Comment (ralfhandl) [inline vocabularies/Hierarchy.xml:163]: Adjust text to match property name

**Review by ralfhandl — CHANGES_REQUESTED** [review]

**Review by ralfhandl — COMMENTED** [review]
If these actions are RAP-specific, should we move them to a RAP vocabulary?

Or are we confident that CAP backends would also be able to implement them?

File: vocabularies/Hierarchy.xml [hunk 3]
```diff
@@ -256,184 +242,72 @@ The template actions themselves cannot be invoked.
...
+      <Action Name="Template_LinkAction" IsBound="true">
         <Annotation Term="Common.Experimental" />
-        <Annotation Term="Core.Description" String="Template for instance-bound actions that create a new non-root node and are named in [`RecursiveHierarchy/CreateChildAction`](#RecursiveHierarchyType)" />
+        <Annotation Term="Core.Description" String="Template for actions that link a node to a parent and are named in [`RecursiveHierarchy/LinkAction`](#RecursiveHierarchyType)" />
         <Annotation Term="Core.LongDescription">
-          <String>This action can be used when creating several nodes in one JSON batch request like:
+          <String>In single-parent hierarchies, the node is first unlinked from its current parent, if any.
+
+To create a new node as child of an existing parent, this action can be used in a JSON batch request like:
 ```
```
Comment (ralfhandl) [inline vocabularies/Hierarchy.xml:252]:
```suggestion
```json
```

Comment (HeikoTheissen) [reply vocabularies/Hierarchy.xml:252]: Not valid JSON, because the code contains ...

**Comment by HeikoTheissen** [comment]
> If these actions are RAP-specific, should we move them to a RAP vocabulary?

No, because the properties `RecursiveHierarchy/LinkAction` etc. are optional anyway, CAP backends could omit them (and perhaps use other optional CAP-specific properties).

**Pull request closed without merging**